edge-agent/docker/linux/amd64/Dockerfile

```diff
@@ -1,7 +1,7 @@
-ARG base_tag=2.1.6-runtime-alpine3.7
+ARG base_tag=2.1.10-alpine3.7
```

Comment:
Should this keep the `runtime` part of the tag? I want to make sure we aren't including the dotnet sdk in the image.

Reply:
dotnet images are now pushed to mcr and the name changed to mcr.microsoft.com/dotnet/core/runtime for runtime and mcr.microsoft.com/dotnet/core/sdk for sdk

Reply:
ah ok cool